File: Java for Clicker/app/src/main/java/com/example/clickergame/MainActivity.java
package com.example.clickergame;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    int count = 0;
    Button up = null;
    Button down = null;
    TextView result = null;
    ImageView photo = null;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        up = (Button) findViewById(R.id.up_button);
        down = (Button) findViewById(R.id.down_button2);
        result = (TextView) findViewById(R.id.result);
        photo = (ImageView) findViewById(R.id.photo);

        View.OnClickListener clickListener = new View.OnClickListener() {


            @Override

            public void onClick(View v)

            {
                switch (v.getId()) {

                    case R.id.up_button:
                        count++;
                        result.setText(""+count);
                        break;

                    case R.id.down_button2:
                        count--;
                        result.setText(""+count);
                        break;


            }
            if(count == 10) {
                Toast.makeText(MainActivity.this, "10점에 도달하였습니다.",Toast.LENGTH_SHORT).show();
                photo.setImageResource(R.drawable.smallchicken);
            }

            else if(count == 20) {
                Toast.makeText(MainActivity.this, "20점에 도달하였습니다!",Toast.LENGTH_SHORT).show();
                photo.setImageResource(R.drawable.chicken);
            }

            else if(count == 30) {
                Toast.makeText(MainActivity.this, "30점에 도달하였습니다!",Toast.LENGTH_SHORT).show();
                photo.setImageResource(R.drawable.chicken);
            }

            }
        };

        up.setOnClickListener(clickListener);
        down.setOnClickListener(clickListener);
    }
}

File: Kotlin for Clicker/app/src/main/java/com/example/kotlinforclicker/MainActivity.kt
package com.example.kotlinforclicker

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


        var count = 0
        var result = findViewById<TextView>(R.id.result)
        var up = findViewById<Button>(R.id.up_button)
        var down = findViewById<Button>(R.id.down_button2)

        up.setOnClickListener{
            count++
            result.text = count.toString()

            if(count == 10) {
                var t1 = Toast.makeText(this,"10점에 도달했습니다.",Toast.LENGTH_SHORT)
                t1.show()

            }

            else if(count == 20) {
                var t2 = Toast.makeText(this,"20점에 도달했습니다.",Toast.LENGTH_SHORT)
                t2.show()

            }
        }
        down.setOnClickListener {
            count--
            result.text = count.toString()
        }

    }
}